--- plasma-workspace-5.11.0/startkde/startkde.cmake.0000~	2017-10-05 14:53:30.000000000 +0200
+++ plasma-workspace-5.11.0/startkde/startkde.cmake	2017-10-06 01:19:56.335037338 +0200
@@ -3,6 +3,11 @@
 #  DEFAULT Plasma STARTUP SCRIPT ( @PROJECT_VERSION@ )
 #
 
+LIBDIR=@LIBDIR@
+
+export PATH=/usr/$LIBDIR/qt5/bin:$PATH
+export QML2_IMPORT_PATH=/usr/$LIBDIR/qml:/usr/$LIBDIR/qt5/qml:$QML2_IMPORT_PATH
+
 # When the X server dies we get a HUP signal from xinit. We must ignore it
 # because we still need to do some cleanup.
 trap 'echo GOT SIGHUP' HUP
@@ -42,15 +47,12 @@ fi
 
 mkdir -p $configDir
 
-#This is basically setting defaults so we can use them with kstartupconfig5
-cat >$configDir/startupconfigkeys <<EOF
-kcminputrc Mouse cursorTheme 'breeze_cursors'
-kcminputrc Mouse cursorSize ''
-ksplashrc KSplash Theme Breeze
-ksplashrc KSplash Engine KSplashQML
-kdeglobals KScreen ScreenScaleFactors ''
-kcmfonts General forceFontDPI 0
-EOF
+kdehome=$HOME/@KDE_DEFAULT_HOME@
+test -n "$KDEHOME" && kdehome=`echo "$KDEHOME"|sed "s,^~/,$HOME/,"`
+# see kstartupconfig source for usage
+mkdir -m 700 -p $kdehome
+mkdir -m 700 -p $kdehome/share
+mkdir -m 700 -p $kdehome/share/config
 
 # preload the user's locale on first start
 plasmalocalerc=$configDir/plasma-localerc
@@ -71,12 +73,12 @@ test -f $exportformatssettings && {
 # Write a default kdeglobals file to set up the font
 kdeglobalsfile=$configDir/kdeglobals
 test -f $kdeglobalsfile || {
-cat >$kdeglobalsfile <<EOF
-[General]
-XftAntialias=true
-XftHintStyle=hintmedium
-XftSubPixel=none
-EOF
+cp -f /etc/xdg/kdeglobals $kdeglobalsfile
+}
+
+startupconfigkeysfile=$configDir/startupconfigkeys
+test -f $startupconfigkeysfile || {
+cp -f /etc/xdg/startupconfigkeys $configDir/startupconfigkeys
 }
 
 kstartupconfig5
--- plasma-workspace-5.11.0/startkde/startplasmacompositor.cmake.0000~	2017-10-05 14:53:30.000000000 +0200
+++ plasma-workspace-5.11.0/startkde/startplasmacompositor.cmake	2017-10-06 01:23:06.071582760 +0200
@@ -3,6 +3,10 @@
 #  DEFAULT Plasma STARTUP SCRIPT ( @PROJECT_VERSION@ )
 #
 
+LIBDIR=@LIBDIR@
+export PATH=/usr/$LIBDIR/qt5/bin:$PATH
+export QML2_IMPORT_PATH=/usr/$LIBDIR/qml:/usr/$LIBDIR/qt5/qml:$QML2_IMPORT_PATH
+
 # in case we have been started with full pathname spec without being in PATH
 bindir=`echo "$0" | sed -n 's,^\(/.*\)/[^/][^/]*$,\1,p'`
 if [ -n "$bindir" ]; then
@@ -25,14 +29,11 @@ fi
 
 mkdir -p $configDir
 
-#This is basically setting defaults so we can use them with kstartupconfig5
-cat >$configDir/startupconfigkeys <<EOF
-kcminputrc Mouse cursorTheme 'breeze_cursors'
-kcminputrc Mouse cursorSize ''
-ksplashrc KSplash Theme Breeze
-ksplashrc KSplash Engine KSplashQML
-kcmfonts General forceFontDPIWayland 0
-EOF
+kdehome=$HOME/@KDE_DEFAULT_HOME@
+test -n "$KDEHOME" && kdehome=`echo "$KDEHOME"|sed "s,^~/,$HOME/,"`
+mkdir -m 700 -p $kdehome
+mkdir -m 700 -p $kdehome/share
+mkdir -m 700 -p $kdehome/share/config
 
 # preload the user's locale on first start
 plasmalocalerc=$configDir/plasma-localerc
@@ -53,12 +54,12 @@ test -f $exportformatssettings && {
 # Write a default kdeglobals file to set up the font
 kdeglobalsfile=$configDir/kdeglobals
 test -f $kdeglobalsfile || {
-cat >$kdeglobalsfile <<EOF
-[General]
-XftAntialias=true
-XftHintStyle=hintmedium
-XftSubPixel=none
-EOF
+cp -f /etc/xdg/kdeglobals $kdeglobalsfile
+}
+
+startupconfigkeysfile=$configDir/startupconfigkeys
+test -f $startupconfigkeysfile || {
+cp -f /etc/xdg/startupconfigkeys $configDir/startupconfigkeys
 }
 
 # Make sure the Oxygen font is installed
