--- plasma-workspace-5.1.1/startkde/startkde.cmake.omv~	2014-11-14 00:23:32.031091615 +0100
+++ plasma-workspace-5.1.1/startkde/startkde.cmake	2014-11-14 00:24:48.018346381 +0100
@@ -3,6 +3,9 @@
 #  DEFAULT KDE STARTUP SCRIPT ( @PROJECT_VERSION@ )
 #
 
+export PATH=/usr/lib64/qt5/bin:$PATH
+export QML2_IMPORT_PATH=/usr/lib64/qml:/usr/lib64/qt5/qml:$QML2_IMPORT_PATH
+
 if test "x$1" = x--failsafe; then
     KDE_FAILSAFE=1 # General failsafe flag
     KWIN_COMPOSE=N # Disable KWin's compositing
