diff -Naur plasma-workspace-5.4.0/startkde/startkde.cmake plasma-workspace-5.4.0.tpg/startkde/startkde.cmake
--- plasma-workspace-5.4.0/startkde/startkde.cmake	2015-08-22 10:45:41.000000000 +0000
+++ plasma-workspace-5.4.0.tpg/startkde/startkde.cmake	2015-09-03 23:37:02.093367041 +0000
@@ -3,6 +3,15 @@
 #  DEFAULT KDE STARTUP SCRIPT ( @PROJECT_VERSION@ )
 #
 
+if [ arch != "i586" ]; then
+    LIBDIR=lib64
+else
+    LIBDIR=lib
+fi
+
+export PATH=/usr/$LIBDIR/qt5/bin:$PATH
+export QML2_IMPORT_PATH=/usr/$LIBDIR/qml:/usr/$LIBDIR/qt5/qml:$QML2_IMPORT_PATH
+
 if test "x$1" = x--failsafe; then
     KDE_FAILSAFE=1 # General failsafe flag
     KWIN_COMPOSE=N # Disable KWin's compositing
@@ -63,16 +72,18 @@
 fi
 
 mkdir -p $configDir
+kdehome=$HOME/@KDE_DEFAULT_HOME@
+test -n "$KDEHOME" && kdehome=`echo "$KDEHOME"|sed "s,^~/,$HOME/,"`
+
+# see kstartupconfig source for usage
+mkdir -m 700 -p $kdehome
+mkdir -m 700 -p $kdehome/share
+mkdir -m 700 -p $kdehome/share/config
 
 #This is basically setting defaults so we can use them with kstartupconfig5
-cat >$configDir/startupconfigkeys <<EOF
-kcminputrc Mouse cursorTheme 'breeze_cursors'
-kcminputrc Mouse cursorSize ''
-ksplashrc KSplash Theme Breeze
-ksplashrc KSplash Engine KSplashQML
-kdeglobals KScreen ScaleFactor 1
-kcmfonts General forceFontDPI 0
-EOF
+for files in kcmdisplayrc kcmfonts kcminputrc kdeglobals kscreenlockerrc ksplashrc kwinrc plasmarc startupconfig startupconfigfiles startupconfigkeys; do
+    cp -f /etc/xdg/$files $configDir/$files ;
+done
 
 # preload the user's locale on first start
 plasmalocalerc=$configDir/plasma-localerc
@@ -90,17 +101,6 @@
     . $exportformatssettings
 }
 
-# Write a default kdeglobals file to set up the font
-kdeglobalsfile=$configDir/kdeglobals
-test -f $kdeglobalsfile || {
-cat >$kdeglobalsfile <<EOF
-[General]
-XftAntialias=true
-XftHintStyle=hintmedium
-XftSubPixel=none
-EOF
-}
-
 # Make sure the Oxygen font is installed
 # This is necessary for setups where CMAKE_INSTALL_PREFIX
 # is not in /usr. fontconfig looks in /usr, ~/.fonts and
