--- plasma-workspace-5.7.0/startkde/startkde.cmake.0000~	2016-06-30 18:33:30.000000000 +0200
+++ plasma-workspace-5.7.0/startkde/startkde.cmake	2016-07-04 13:51:46.369601658 +0200
@@ -3,6 +3,11 @@
 #  DEFAULT KDE STARTUP SCRIPT ( @PROJECT_VERSION@ )
 #
 
+LIBDIR=@LIBDIR@
+
+export PATH=/usr/$LIBDIR/qt5/bin:$PATH
+export QML2_IMPORT_PATH=/usr/$LIBDIR/qml:/usr/$LIBDIR/qt5/qml:$QML2_IMPORT_PATH
+
 if test "x$1" = x--failsafe; then
     KDE_FAILSAFE=1 # General failsafe flag
     KWIN_COMPOSE=N # Disable KWin's compositing
@@ -64,16 +69,12 @@ else
 fi
 
 mkdir -p $configDir
-
-#This is basically setting defaults so we can use them with kstartupconfig5
-cat >$configDir/startupconfigkeys <<EOF
-kcminputrc Mouse cursorTheme 'breeze_cursors'
-kcminputrc Mouse cursorSize ''
-ksplashrc KSplash Theme Breeze
-ksplashrc KSplash Engine KSplashQML
-kdeglobals KScreen ScreenScaleFactors ''
-kcmfonts General forceFontDPI 0
-EOF
+kdehome=$HOME/@KDE_DEFAULT_HOME@
+test -n "$KDEHOME" && kdehome=`echo "$KDEHOME"|sed "s,^~/,$HOME/,"`
+# see kstartupconfig source for usage
+mkdir -m 700 -p $kdehome
+mkdir -m 700 -p $kdehome/share
+mkdir -m 700 -p $kdehome/share/config
 
 # preload the user's locale on first start
 plasmalocalerc=$configDir/plasma-localerc
@@ -91,17 +92,6 @@ test -f $exportformatssettings && {
     . $exportformatssettings
 }
 
-# Write a default kdeglobals file to set up the font
-kdeglobalsfile=$configDir/kdeglobals
-test -f $kdeglobalsfile || {
-cat >$kdeglobalsfile <<EOF
-[General]
-XftAntialias=true
-XftHintStyle=hintmedium
-XftSubPixel=none
-EOF
-}
-
 # Make sure the Oxygen font is installed
 # This is necessary for setups where CMAKE_INSTALL_PREFIX
 # is not in /usr. fontconfig looks in /usr, ~/.fonts and
