diff -Naur plasma-workspace-5.5.3/startkde/startplasmacompositor.cmake plasma-workspace-5.5.3.tpg/startkde/startplasmacompositor.cmake
--- plasma-workspace-5.5.3/startkde/startplasmacompositor.cmake	2016-01-06 12:17:36.000000000 +0000
+++ plasma-workspace-5.5.3.tpg/startkde/startplasmacompositor.cmake	2016-01-18 00:16:16.583392343 +0000
@@ -3,6 +3,15 @@
 #  DEFAULT KDE STARTUP SCRIPT ( @PROJECT_VERSION@ )
 #
 
+if [ arch != "i586" ]; then
+    LIBDIR=lib64
+else
+    LIBDIR=lib
+fi
+
+export PATH=/usr/$LIBDIR/qt5/bin:$PATH
+export QML2_IMPORT_PATH=/usr/$LIBDIR/qml:/usr/$LIBDIR/qt5/qml:$QML2_IMPORT_PATH
+
 # in case we have been started with full pathname spec without being in PATH
 bindir=`echo "$0" | sed -n 's,^\(/.*\)/[^/][^/]*$,\1,p'`
 if [ -n "$bindir" ]; then
@@ -24,16 +33,13 @@
 fi
 
 mkdir -p $configDir
+kdehome=$HOME/@KDE_DEFAULT_HOME@
+test -n "$KDEHOME" && kdehome=`echo "$KDEHOME"|sed "s,^~/,$HOME/,"`
 
-#This is basically setting defaults so we can use them with kstartupconfig5
-cat >$configDir/startupconfigkeys <<EOF
-kcminputrc Mouse cursorTheme 'breeze_cursors'
-kcminputrc Mouse cursorSize ''
-ksplashrc KSplash Theme Breeze
-ksplashrc KSplash Engine KSplashQML
-kdeglobals KScreen ScaleFactor 1
-kcmfonts General forceFontDPI 0
-EOF
+# see kstartupconfig source for usage
+mkdir -m 700 -p $kdehome
+mkdir -m 700 -p $kdehome/share
+mkdir -m 700 -p $kdehome/share/config
 
 # preload the user's locale on first start
 plasmalocalerc=$configDir/plasma-localerc
@@ -51,17 +57,6 @@
     . $exportformatssettings
 }
 
-# Write a default kdeglobals file to set up the font
-kdeglobalsfile=$configDir/kdeglobals
-test -f $kdeglobalsfile || {
-cat >$kdeglobalsfile <<EOF
-[General]
-XftAntialias=true
-XftHintStyle=hintmedium
-XftSubPixel=none
-EOF
-}
-
 # Make sure the Oxygen font is installed
 # This is necessary for setups where CMAKE_INSTALL_PREFIX
 # is not in /usr. fontconfig looks in /usr, ~/.fonts and
