diff -Naur plasma-workspace-5.9.0/ksplash/ksplashqml/themes/Classic/main.qml plasma-workspace-5.9.0.tpg/ksplash/ksplashqml/themes/Classic/main.qml
--- plasma-workspace-5.9.0/ksplash/ksplashqml/themes/Classic/main.qml	2017-01-26 12:28:32.000000000 +0000
+++ plasma-workspace-5.9.0.tpg/ksplash/ksplashqml/themes/Classic/main.qml	2017-02-01 21:00:05.977659905 +0000
@@ -52,7 +52,7 @@
     Image {
         id: background
         anchors.fill: parent
-        source: "images/background.png"
+        source: "/usr/share/mdk/backgrounds/OpenMandriva-splash.png"
     }
 
     Image {
diff -Naur plasma-workspace-5.9.0/lookandfeel/contents/splash/Splash.qml plasma-workspace-5.9.0.tpg/lookandfeel/contents/splash/Splash.qml
--- plasma-workspace-5.9.0/lookandfeel/contents/splash/Splash.qml	2017-01-26 12:28:32.000000000 +0000
+++ plasma-workspace-5.9.0.tpg/lookandfeel/contents/splash/Splash.qml	2017-02-01 21:04:01.266098291 +0000
@@ -19,9 +19,10 @@
 import QtQuick 2.5
 import QtQuick.Window 2.2
 
-Rectangle {
+Image {
     id: root
-    color: "black"
+    source: "../components/artwork/background.png"
+    fillMode: Image.PreserveAspectCrop
 
     property int stage
 
@@ -55,7 +56,7 @@
 
             anchors.centerIn: parent
 
-            source: "images/plasma.svgz"
+            source: "/usr/share/icons/openmandriva.svg"
 
             sourceSize.width: size
             sourceSize.height: size
