diff -Naur plasma-workspace-5.5.2/lookandfeel/contents/splash/Splash.qml plasma-workspace-5.5.2.tpg/lookandfeel/contents/splash/Splash.qml
--- plasma-workspace-5.5.2/lookandfeel/contents/splash/Splash.qml	2015-12-22 11:01:15.000000000 +0000
+++ plasma-workspace-5.5.2.tpg/lookandfeel/contents/splash/Splash.qml	2016-01-07 01:12:58.799681508 +0000
@@ -38,7 +38,7 @@
         y: root.height
         color: "#4C000000"
         Image {
-            source: "images/kde.svgz"
+            source: "/usr/share/icons/openmandriva.svg"
             anchors.centerIn: parent
             sourceSize.height: 128
             sourceSize.width: 128
