diff -Naur plasma-workspace-5.18.3/ksplash/ksplashqml/themes/Classic/main.qml plasma-workspace-5.18.3.tpg/ksplash/ksplashqml/themes/Classic/main.qml
--- plasma-workspace-5.18.3/ksplash/ksplashqml/themes/Classic/main.qml	2020-03-10 13:24:06.000000000 +0000
+++ plasma-workspace-5.18.3.tpg/ksplash/ksplashqml/themes/Classic/main.qml	2020-03-14 21:55:47.465592926 +0000
@@ -52,7 +52,7 @@
     Image {
         id: background
         anchors.fill: parent
-        source: "images/background.png"
+        source: "/usr/share/mdk/backgrounds/default.png"
     }
 
     Image {
diff -Naur plasma-workspace-5.18.3/lookandfeel/contents/components/WallpaperFader.qml plasma-workspace-5.18.3.tpg/lookandfeel/contents/components/WallpaperFader.qml
--- plasma-workspace-5.18.3/lookandfeel/contents/components/WallpaperFader.qml	2020-03-10 13:24:06.000000000 +0000
+++ plasma-workspace-5.18.3.tpg/lookandfeel/contents/components/WallpaperFader.qml	2020-03-14 21:57:45.245508163 +0000
@@ -115,11 +115,11 @@
             }
             PropertyChanges {
                 target: wallpaperFader
-                factor: 1
+                factor: 0
             }
             PropertyChanges {
                 target: clock.shadow
-                opacity: 0
+                opacity: 1
             }
             PropertyChanges {
                 target: clock
diff -Naur plasma-workspace-5.18.3/lookandfeel/contents/splash/Splash.qml plasma-workspace-5.18.3.tpg/lookandfeel/contents/splash/Splash.qml
--- plasma-workspace-5.18.3/lookandfeel/contents/splash/Splash.qml	2020-03-10 13:24:06.000000000 +0000
+++ plasma-workspace-5.18.3.tpg/lookandfeel/contents/splash/Splash.qml	2020-03-14 21:55:47.465592926 +0000
@@ -20,9 +20,10 @@
 import QtQuick 2.5
 import QtQuick.Window 2.2
 
-Rectangle {
+Image {
     id: root
-    color: "black"
+    source: "../components/artwork/background.png"
+    fillMode: Image.PreserveAspectCrop
 
     property int stage
 
@@ -56,7 +57,7 @@
 
             anchors.centerIn: parent
 
-            source: "images/plasma.svgz"
+            source: "/usr/share/icons/openmandriva.svg"
 
             sourceSize.width: size
             sourceSize.height: size
diff -Naur plasma-workspace-5.18.3/sddm-theme/components/WallpaperFader.qml plasma-workspace-5.18.3.tpg/sddm-theme/components/WallpaperFader.qml
--- plasma-workspace-5.18.3/sddm-theme/components/WallpaperFader.qml	2020-03-10 13:24:06.000000000 +0000
+++ plasma-workspace-5.18.3.tpg/sddm-theme/components/WallpaperFader.qml	2020-03-14 21:57:45.245508163 +0000
@@ -115,11 +115,11 @@
             }
             PropertyChanges {
                 target: wallpaperFader
-                factor: 1
+                factor: 0
             }
             PropertyChanges {
                 target: clock.shadow
-                opacity: 0
+                opacity: 1
             }
             PropertyChanges {
                 target: clock
