diff -up plasma-workspace-5.13.4/ksplash/ksplashqml/themes/Classic/main.qml.0002~ plasma-workspace-5.13.4/ksplash/ksplashqml/themes/Classic/main.qml
--- plasma-workspace-5.13.4/ksplash/ksplashqml/themes/Classic/main.qml.0002~	2018-07-31 14:57:52.000000000 +0200
+++ plasma-workspace-5.13.4/ksplash/ksplashqml/themes/Classic/main.qml	2018-07-31 18:05:04.924510151 +0200
@@ -52,7 +52,7 @@ Item {
     Image {
         id: background
         anchors.fill: parent
-        source: "images/background.png"
+        source: "/usr/share/mdk/backgrounds/OpenMandriva-splash.png"
     }
 
     Image {
diff -up plasma-workspace-5.13.4/lookandfeel/contents/splash/Splash.qml.0002~ plasma-workspace-5.13.4/lookandfeel/contents/splash/Splash.qml
--- plasma-workspace-5.13.4/lookandfeel/contents/splash/Splash.qml.0002~	2018-07-31 14:57:52.000000000 +0200
+++ plasma-workspace-5.13.4/lookandfeel/contents/splash/Splash.qml	2018-07-31 18:10:46.934066186 +0200
@@ -20,9 +20,10 @@
 import QtQuick 2.5
 import QtQuick.Window 2.2
 
-Rectangle {
+Image {
     id: root
-    color: "black"
+    source: "../components/artwork/background.png"
+    fillMode: Image.PreserveAspectCrop
 
     property int stage
 
@@ -56,7 +57,7 @@ Rectangle {
 
             anchors.centerIn: parent
 
-            source: "images/plasma.svgz"
+            source: "/usr/share/icons/openmandriva.svg"
 
             sourceSize.width: size
             sourceSize.height: size
