diff -Naur plasma-workspace-5.8.0/ksplash/ksplashqml/themes/Classic/main.qml plasma-workspace-5.8.0.tpg/ksplash/ksplashqml/themes/Classic/main.qml
--- plasma-workspace-5.8.0/ksplash/ksplashqml/themes/Classic/main.qml	2016-09-29 11:35:01.000000000 +0000
+++ plasma-workspace-5.8.0.tpg/ksplash/ksplashqml/themes/Classic/main.qml	2016-10-12 18:27:36.185822778 +0000
@@ -52,7 +52,7 @@
     Image {
         id: background
         anchors.fill: parent
-        source: "images/background.png"
+        source: "/usr/share/mdk/backgrounds/OpenMandriva-splash.png"
     }
 
     Image {
diff -Naur plasma-workspace-5.8.0/lookandfeel/contents/splash/Splash.qml plasma-workspace-5.8.0.tpg/lookandfeel/contents/splash/Splash.qml
--- plasma-workspace-5.8.0/lookandfeel/contents/splash/Splash.qml	2016-09-29 11:35:01.000000000 +0000
+++ plasma-workspace-5.8.0.tpg/lookandfeel/contents/splash/Splash.qml	2016-10-12 18:34:02.049807468 +0000
@@ -55,7 +55,7 @@
 
             anchors.centerIn: parent
 
-            source: "images/kde.svgz"
+            source: "/usr/share/icons/openmandriva.svg"
 
             sourceSize.width: size
             sourceSize.height: size
