diff -Naur plasma-workspace-5.8.2/ksplash/ksplashqml/themes/Classic/main.qml plasma-workspace-5.8.2.tpg/ksplash/ksplashqml/themes/Classic/main.qml
--- plasma-workspace-5.8.2/ksplash/ksplashqml/themes/Classic/main.qml	2016-10-18 11:47:17.000000000 +0000
+++ plasma-workspace-5.8.2.tpg/ksplash/ksplashqml/themes/Classic/main.qml	2016-10-27 00:40:51.992542329 +0000
@@ -52,7 +52,7 @@
     Image {
         id: background
         anchors.fill: parent
-        source: "images/background.png"
+        source: "/usr/share/mdk/backgrounds/OpenMandriva-splash.png"
     }
 
     Image {
diff -Naur plasma-workspace-5.8.2/lookandfeel/contents/splash/Splash.qml plasma-workspace-5.8.2.tpg/lookandfeel/contents/splash/Splash.qml
--- plasma-workspace-5.8.2/lookandfeel/contents/splash/Splash.qml	2016-10-18 11:47:17.000000000 +0000
+++ plasma-workspace-5.8.2.tpg/lookandfeel/contents/splash/Splash.qml	2016-10-27 00:42:36.780827144 +0000
@@ -19,9 +19,10 @@
 
 import QtQuick 2.5
 
-Rectangle {
+Image {
     id: root
-    color: "black"
+    source: "../components/artwork/background.png"
+    fillMode: Image.PreserveAspectCrop
 
     property int stage
 
@@ -55,7 +56,7 @@
 
             anchors.centerIn: parent
 
-            source: "images/kde.svgz"
+            source: "/usr/share/icons/openmandriva.svg"
 
             sourceSize.width: size
             sourceSize.height: size
